ui/app/AppLayouts/Chat/popups/community/CreateCategoryPopup.qml

```diff
@@ -119,6 +119,11 @@ StatusModal {
                         visible: model.type != Constants.chatType.unknown
                         height: visible ? implicitHeight : 0
                         title: "#" + model.name
+                        icon.emoji: model.emoji
+                        icon.color: model.color
+                        image.isIdenticon: false
+                        image.source: model.icon
+                        ringSettings.ringSpecModel: model.colorHash
```

Comment:
There will never be a colorHash for those types of channels, but it's not a big deal, since `colorHash` is null here.